changelog.d/4-docs/sqservices-1127

```diff
@@ -0,0 +1 @@
+Improve documentation for several test cases per SQSERVICES-1127
```

Comment:
The changelog's target audience is the public, so I would like to avoid internal jira issues.

Reply:
A fair point. I updated the change log so it does not point to the ticket.